spec/Cabin/Infotainment.vspec

```diff
@@ -102,41 +102,41 @@ HMI.CurrentLanguage:
 HMI.DateFormat:
   datatype: string
   type: actuator
-  allowed: [ "YYYY MM DD", "DD MM YYYY", "MM DD YYYY", "YY MM DD", "DD MM YY", "MM DD YY"]
+  allowed: [YYYY_MM_DD, DD_MM_YYYY, MM_DD_YYYY, YY_MM_DD, DD_MM_YY, MM_DD_YY]
   description: Date format used in the current HMI
 
 HMI.TimeFormat:
   datatype: string
   type: actuator
-  allowed: [ "12HR", "24HR" ]
+  allowed: [FORMAT_12_HR, FORMAT_24_HR]
```

Comment:
My preference is that it does not need to be more verbose.  (Do not add FORMAT_)

Also, not sure if we are 100% through the discussion of allowed/enum, but my interpretation here is that normally the _string_ will be transferred, so in that case this is just increasing the bandwidth usage for string-transfer for no reason?

Reply:
@erikbosch changes it to HR_x and then it can be merged.